### PR TITLE
Add support for PHP8 attribute

### PR DESCRIPTION
**Pull request by ker0x**

### CODE REVIEW

**Comment by Nyholm** [comment]
Awesome. Thank you. Can you see why the test is failing?

File: tests/EntityExistValidatorTest.php [hunk 1]
```diff
@@ -158,4 +160,41 @@ public function testValidateInvalidEntity(): void
...
+
+    /**
+     * @requires PHP 8
+     */
+    public function testValidateFromAttribute()
+    {
+        $this->context->expects($this->never())->method('buildViolation');
+
+        $classMetadata = new ClassMetadata(EntityDummy::class);
+        (new AnnotationLoader())->loadClassMetadata($classMetadata);
```
Comment (ker0x) [inline tests/EntityExistValidatorTest.php:172]: @Nyholm the AnnotationLoader class expect a Reader argument for Symfony version < 5.2. That's why the test is failling.

Comment (Nyholm) [reply tests/EntityExistValidatorTest.php:172]: Interesting. 
Is there a workaround or should we drop SF <5.2?

**Comment by louisezetterlund** [comment]
Hi! Is this still being worked on? 😊

**Comment by Nyholm** [comment]
I’m not currently working on it. 

@louisezetterlund would you like to continue the work?

**Pull request closed without merging**